.github/workflows/copilot-setup-steps.yml

```
@@ -1,266 +1,266 @@
-
-              $dscResources = @(dsc resource list --adapter Microsoft.DSC/PowerShell --output-format json | ConvertFrom-Json)
-              Write-Host "`n  Available class-based DSC Resources: $($dscResources.Count)`n"
-              $dscResources | Format-Table type, capabilities, implementedAs, requireAdapter -GroupBy kind -AutoSize
-          }
-          else
-          {
-              Write-Error 'Failed to import module'
-
-              exit 1
-          }
+name: 'Copilot Setup Steps'
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)